content/references/sdk.md

```diff
@@ -450,6 +528,102 @@ Note: `page` will be used if both `page` and `offset` are present.
 ]
 ```
 
+#### 'products.getByCategorySlug()'
```

Comment:
Perhaps we should wait to add this to the docs until it can support the sort methods since this will be the most popular method of getting product data on category pages.

Reply:
Our next sprint is going to be 4 days long, so no guarantee when that gets added. Actually, if Timbo hasn't finished adding sorting to the endpoint before sprint planning there's a strong chance it will go into the next sprint. My recommendation would be to document what we've got for now.